app/models/adopter.rb

```diff
@@ -118,6 +120,10 @@ def attributes_to_audit
     %w[status assigned_to_user_id email phone address1 address2 city state zip]
   end
 
+  def audits_and_associated_audits
+    (audits + associated_audits).sort_by(&:created_at)
```

Comment:
@TheDudeWithTheThing any suggestions on how to sort this stuff?

Reply:
This is what I tried too, it works on console but not in the app right? So strange, I have a theory

Reply:
Think I got it working, change pushed